internal/pkg/log/cli.go

```diff
@@ -0,0 +1,27 @@
+package log
```

Comment:
commit `Add log.Logger and log.DebugLogger interfaces` 

------------------

Vid najprv subor `log.go` nizsie.